static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -185,6 +186,20 @@ export function fetchDateList(
   });
 }
 
+function parseEventPropVal(propVal, unit: string, reqParams?: any): number {
+  if (!_.isEmpty(propVal)) {
+    // Get value by taking the first severity val, trimming the unit
```

Comment:
update comments - no longer just for severity. Also the console.assert message should be updated

Reply:
thanks, done